src/Enums/StructureResolverSort.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Spatie\StructureDiscoverer\Enums;
+
+use Symfony\Component\Finder\Finder;
+
+enum StructureResolverSort
+{
+    case NAME;
+    case SIZE;
+    case TYPE;
+    case EXTENSION;
+    case CHANGED_TIME;
```

Comment:
We use `PascalCase` for enums. Could you change that. If you need to have a value with another casing, feel free to add string values to this enum.